geode-core/src/main/java/org/apache/geode/internal/cache/InternalCacheForClientAccess.java

```diff
@@ -1215,6 +1216,14 @@ public InternalQueryService getInternalQueryService() {
     return delegate.getInternalQueryService();
   }
 
+  @Override
+  public boolean hasMemberOlderThan(Version version) {
```

Comment:
You don't have to create a new method in GemFireCacheImpl. 

LuceneServiceImpl has the validateAllMembersAreTheSameVersion()